src/main/java/org/isf/medicalinventory/model/MedicalInventory.java

```diff
@@ -148,4 +148,12 @@ public String getWard() {
 	public void setWard(String ward) {
 		this.ward = ward;
 	}
+
+	public int getLock() {
+		return lock;
+	}
+
+	public void setLock(int lock) {
+		this.lock = lock;
+	}
```

Comment:
This should be already landed on core, right?

Reply:
No; this is part of https://github.com/informatici/openhospital-core/pull/1300